xfce4-settings-qubes.spec.in

```diff
@@ -110,7 +111,9 @@ fi
 %{_sysconfdir}/xdg/xfce4/xfconf/xfce-perchannel-xml/xfce4-keyboard-shortcuts.xml.qubes
 %{_sysconfdir}/xdg/autostart/xfce4-xss-lock.desktop
 %{_sysconfdir}/xdg/autostart/qubes-update-xfce-config.desktop
+%{_sysconfdir}/xdg/menus/xfce-applications.menu.qubes
```

Comment:
This is not the case anymore, no?

Reply:
Yes it's wrongly has not been included in the commit. Sorry.